### PR TITLE
Jp dev drop down fix

### PR DESCRIPTION
**Pull request by phillipsja97**
* updated the blocks to show only a few dropdowns that are needed.
* updated the variables to show upon peripheral sign in.

### CODE REVIEW

File: src/extensions/scratch3_playspot/index.js [hunk 1]
```diff
@@ -992,43 +944,18 @@ class Scratch3PlayspotBlocks {
...
-                            menu: 'modes',
-                            defaultValue: 'Unknown'
+                            defaultValue: defaultSatellite
                         }
                     }
                 },
-                '---',
                 {
-                    opcode: 'displayLightSequence',
+                    opcode: 'displayTheLightSequence',
```
Comment (rvsrvs) [inline src/extensions/scratch3_playspot/index.js]: Not sure I understand This name change...

**Review by rvsrvs — COMMENTED** [review]
Just one comment.  It may be ok..

File: src/extensions/scratch3_playspot/index.js (hunk 1 above)
Comment (phillipsja97) [reply src/extensions/scratch3_playspot/index.js]: I can change it back if need be. I was testing something where I created a new block that copied that one and the function to follow and didn't change it back. The corresponding function has the THE inserted as well so everything still works. Just didn't change it back to the original function name without the THE.

Comment (rvsrvs) [reply src/extensions/scratch3_playspot/index.js]: Would prefer if you did, please.

Comment (phillipsja97) [reply src/extensions/scratch3_playspot/index.js]: Fixed

**Pull request merged**